apps/daimo-mobile/src/action/useCreateAccount.ts

```diff
@@ -33,12 +34,23 @@ export function useCreateAccount(
   // On exec, create contract onchain, claiming name.
   const result = rpcHook.deployWallet.useMutation();
   const exec = async () => {
-    if (!keyStatus.pubKeyHex) return;
+    if (
+      !keyStatus.pubKeyHex ||
+      !deviceAttestationKeyStatus.pubKeyHex ||
+      !sanitisedInviteLink
+    ) {
+      console.log(
+        `[CREATE] missing data for useCreateAccount ${keyStatus} ${deviceAttestationKeyStatus} ${sanitisedInviteLink}`
+      );
+      setAS("error", "Missing data");
```

Comment:
in what situation do we end up showing the `Missing data` error?

Reply:
we shouldn't ever; this is not new code, I've just made the error more visible vs. before